kit/brain_board/robot_os.md

```diff
@@ -1,23 +1,30 @@
 ---
 layout: page
-title: Robot OS
+title: Student Robotics OS
 ---
 
-Robot OS
-========
 
-TODO
+# Student Robotics OS
 
-# Updates
+The Raspberry Pi runs a custom operating system that contains the Student Robotics software.
```

Comment:
Suggestion: We should specify here that it's based on ~Raspbian~ Raspberry Pi OS. The tinkerers will care.

```suggestion
The Raspberry Pi runs a custom operating system that contains the Student Robotics software. It's based on [Raspberry Pi OS](https://www.raspberrypi.com/documentation/computers/os.html) (`2023-05-03`), which in turn is based on [Debian Linux](https://www.debian.org/) (bullseye).
```

Reply:
I think we only need to specify that it's based on Raspberry Pi OS, no need to go down the rabbit hole of Linux pedigrees.

Reply:
a4aa897